src/app.typedefs.ts

```diff
@@ -0,0 +1,43 @@
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {
+    street: string;
+    suite: string;
+    city: string;
+    zipcode: string;
+    geo: {
+      lat: string;
+      lng: string;
+    };
+  };
+  phone: string;
+  website: string;
+  company: {
+    name: string;
+    catchPhrase: string;
+    bs: string;
+  };
+}
+
```

Comment:
this interface is super long and contains 3 objects inside of it, maybe you could separate them?

Reply:
upvote ➕